ui/StatusQ/src/StatusQ/Layout/StatusAppNavBar.qml

```diff
@@ -1,195 +1,129 @@
-    property var filterCommunityItem: function(item) { return true; }
+    property real delegateHeight
 
-    signal aboutToUpdateFilteredRegularModel()
-    signal aboutToUpdateFilteredCommunityModel()
+    property alias cameraComponent: cameraItemLoader.sourceComponent
+    property alias profileComponent: profileItemLoader.sourceComponent
 
-    onNavBarProfileButtonChanged: {
-        if (!!navBarProfileButton) {
-            navBarProfileButton.parent = navBarProfileButtonSlot
-        }
+    implicitWidth: 78
+    implicitHeight: layout.implicitHeight
+
+    Rectangle {
+        anchors.fill: parent
+        color: Theme.palette.statusAppNavBar.backgroundColor
     }
 
-    onNavBarCameraButtonChanged: {
-        if (!!navBarCameraButton) {
-            navBarCameraButton.parent = navBarCameraButtonSlot
+    ColumnLayout {
+        id: layout
+
+        anchors {
+            fill: parent
+            topMargin: 48
+            bottomMargin: 24
         }
-    }
 
-    function triggerUpdate(){
-        navBarModel.update()
-    }
+        spacing: d.spacing
 
-    StatusAppNavBarFilterModel {
-        id: navBarModel
+        ListView {
+            id: chatItemsListView
 
-        filterAcceptsItem: filterRegularItem
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            Layout.minimumHeight: root.delegateHeight
+            Layout.preferredHeight: d.implicitListViewHeight(this)
+            Layout.maximumHeight: Layout.preferredHeight
 
-        model: statusAppNavBar.sectionModel
+            clip: true
+            spacing: d.spacing
+            boundsBehavior: contentHeight > height ? Flickable.DragAndOvershootBounds : Flickable.StopAtBounds
+        }
 
-        onAboutToUpdateFilteredModel: {
-            statusAppNavBar.aboutToUpdateFilteredRegularModel()
+        Rectangle {
+            implicitHeight: 1
+            Layout.preferredWidth: d.separatorWidth
+            Layout.alignment: Qt.AlignHCenter
+            color: Theme.palette.directColor7
+
+            visible: chatItemsListView.count && mmunityItemsListView.contentHeight > communityItemsListView.height
         }
 
-        DelegateChooser {
-            id: delegateChooser
-            role: communityTypeRole
-            DelegateChoice { roleValue: communityTypeValue; delegate: communityNavButton }
-            DelegateChoice { delegate: regularNavBarButton }
+        ListView {
+            id: communityItemsListView
+
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            Layout.minimumHeight: root.delegateHeight
+            Layout.preferredHeight: d.implicitListViewHeight(this)
+            Layout.maximumHeight: Layout.preferredHeight
+
+            clip: true
+            spacing: d.spacing
+            boundsBehavior: contentHeight > height ? Flickable.DragAndOvershootBounds : Flickable.StopAtBounds
         }
 
-        delegate: delegateChooser
-    }
+        Rectangle {
```

Comment:
Can we give this guy an id or a comment to say it's a separator?

Reply:
I left it without id because 1) it is not referenced anywhere 2) there are two separators and I wanted to avoid names like `separator1` and `separator2`

Added ids to them anyway cause I came up with better names :D